tds/src/main/java/thredds/core/DatasetManager.java

```diff
@@ -269,7 +270,7 @@ public FeatureDatasetPoint openPointDataset(HttpServletRequest req, HttpServletR
     NetcdfDataset ncd = null;
     try {
       ncd = NetcdfDataset.wrap(ncfile, NetcdfDataset.getDefaultEnhanceMode());
```

Comment:
Hmm... I'm confuzzled. This method is named `openPointDataset` and it returns a `FeatureDatasetPoint`. Given this context, why should this method concern itself with `SIMPLE_GEOMETRY` `FeatureType`s rather than `ANY_POINT` `FeatureType`s?